styles.css

```diff
@@ -289,11 +280,85 @@ textarea:focus {
 /* --------------- m o b i l e ----------------------- */
 
 @media only screen and (max-width: 767px) {
+  header {
+    margin-left: 5%;
+    display: flex;
+    justify-content: space-between;
+    background-color: white;
+    width: 100%;
+    padding-top: 5%;
+  }
+
   .logo h3 {
-    font-weight: 600px;
+    font-weight: 600;
     font-size: 20px;
   }
 
+  /* dropdown menu start */
+  .container {
+    display: inline-block;
+    cursor: pointer;
+    align-items: center;
+  }
+
+  .bar1,
+  .bar2,
+  .bar3 {
+    width: 35px;
+    height: 5px;
+    background-color: #333;
+    margin: 6px 0;
+    transition: 0.4s;
+  }
+
+  .change .bar1 {
+    transform: translate(0, 11px) rotate(-45deg);
+    background-color: #67798e;
+  }
+
+  .change .bar2 {
+    opacity: 0;
+  }
+
+  .change .bar3 {
+    transform: translate(0, -11px) rotate(45deg);
+    background-color: #67798e;
+  }
+
+  .accordion {
+    color: #444;
+    cursor: pointer;
+    width: 30%;
+    border: none;
+    outline: none;
+    font-size: 15px;
+    transition: 0.4s;
+  }
+
+  #panel {
+    padding: 0 18px;
+    display: none;
+    background-color: white;
+    overflow: hidden;
+  }
```

Comment:
- [ ] Currently, your mobile menu isn't identical to the mobile menu of the Figma template you chose. Please, make the mobile menu the full `view height (vh)` of the mobile screen to cover the whole mobile screen. 👍 💻 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/223190468-84c3eb42-cb6b-4ab0-ac89-1fefcdaddb01.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/223190758-554f7c64-ec16-41c2-9dea-fbbe0bd9e712.png)

Reply:
@Gambit142 I tend to disagree on this point, figma has only displayed the mobile menu to show how it should look but not displayed the background. My point of view is that it looks odd to push all the contents down the view only to show THREE links.
So correct me, I am wrong.

Reply:
Hello @ehmaddd , As seen from the template, the idea is to have just the popup on view. Based on your requirement, if any section is clicked, it takes you to the same path or cancels if a random place is clicked 

## View:
![image](https://user-images.githubusercontent.com/50990925/223311583-6dd6bdde-7f55-46ee-8ce3-b2a9afebdc0b.png)